hadoop-ozone/interface-client/src/main/proto/OmClientProtocol.proto

```diff
@@ -97,6 +97,7 @@ enum Type {
 
   ListMultipartUploads = 82;
 
+  // Not used anymore due to HDDS-11251
   ListTrash = 91;
   RecoverTrash = 92;
```

Comment:
Let's add the `// [deprecated = true]` similar to `PurgePaths`

Reply:
Addressed this in the latest commit